import/importer.mjs

```diff
@@ -0,0 +1,53 @@
+import { readFileSync } from 'fs';
+import { spawn } from 'child_process';
+import markdownConverter from "./markdown-converter.mjs";
+
+export default class Importer {
+    static processNotifyTemplate = (filename) => {
+        let rawdata = readFileSync(filename);
+        let contents = JSON.parse(rawdata);
+    
+        contents.forEach(item => {
+            const post = {
+                title: markdownConverter(item.subject),
```

Comment:
Lets skip converting markdown for the title

Reply:
Given the contents of the file:

Given the subject lines are all the same looks like ideally we would parse the title out of the content

I think they all follow the same pattern :

```# A non-medical mask can help protect those around you\r\n\r\n```


We could split on the `\r\n\r\n` and remove the # 

Thoughts?

Reply:
Ended up parsing the body and get the first paragraph of the content as the title (or if it's too lengthy, use the subject)